addons/sourcemod/scripting/nominations_extended.sp

```diff
@@ -225,7 +254,7 @@ public Action Command_Addmap(int client, int args)
 {
 	if (args < 1)
 	{
-		CReplyToCommand(client, "[SNK.SRV] Usage: sm_nominate_addmap <mapname>");
+		CReplyToCommand(client, "[NE] Usage: sm_nominate_addmap <mapname>");

```

Comment:
This prefix should be replaced with `g_szChatPrefix` and preferably stored as a translation